### PR TITLE
New 3d vectors

### PR DESCRIPTION
**Pull request by paolafer**
This PR adapts the code to using a 3-D vector instead of three different coordinates, when a specific vertex or momentum is passed by the user via configuration parameter.

It also changes the way the `AD_HOC` vertex is implemented in the `NEW` geometry, to make it similar to the rest of the geometries.

### CODE REVIEW

File: macros/NEXT_options.config.mac [hunk 1]
```diff
@@ -11,7 +11,7 @@
 ##### VERBOSITIES #####
 /run/verbose 0
 /event/verbose 0
-/tracking/verbose 0
+/tracking/verbose 1
```
Comment (jmalbos) [inline macros/NEXT_options.config.mac]: Is this change on purpose?

File: macros/NEW_geantino.config.mac [hunk 2]
```diff
@@ -15,9 +15,7 @@
 /Generator/SingleParticle/min_energy 1  eV
 /Generator/SingleParticle/max_energy 1 keV
 /Generator/SingleParticle/region AD_HOC
-/Geometry/NextNew/specific_vertex_X   0 mm
-/Geometry/NextNew/specific_vertex_Y   0 mm
-/Geometry/NextNew/specific_vertex_Z 100 mm
+/Geometry/NextNew/specific_vertex   0 0 100 mm
```
Comment (jmalbos) [inline macros/NEW_geantino.config.mac]: Remove the extra whitespaces between `specific_vertex` and the first `0`.

File: SConstruct [hunk 3]
```diff
@@ -325,7 +324,7 @@ env.Execute(Chmod(w_prefix_dir+'/bin/nexus-config', 0o755))
 nexus = env.Program('bin/nexus', ['source/nexus.cc']+src)
 
 TSTDIR = ['utils',
-	  'example']
+	      'example']
```
Comment (jmalbos) [inline SConstruct]: The alignment has been changed.

Comment (paolafer) [reply SConstruct]: I'm not sure how to fix it, because in my editor it appears as aligned.

File: macros/NEXT_options.config.mac (hunk 1 above)
Comment (paolafer) [reply macros/NEXT_options.config.mac]: No, fixed.

File: SConstruct (hunk 3 above)
Comment (paolafer) [reply SConstruct]: I fixed it using another editor.

**Review by jmalbos — APPROVED** [review]
This PR introduces the use of 3-vectors (plus unit) in the configuration macros. The code compiles and runs as intended. I approve it.
